src/components/PeopleTable/PeopleTable.tsx

```diff
@@ -0,0 +1,76 @@
+import { FC } from 'react';
+import { useMatch } from 'react-router-dom';
+import cn from 'classnames';
+import { Person } from '../../types';
+import { PersonLink } from '../PersonLink';
+
+type Props = {
+  people: Person[]
+};
+
+export const PeopleTable: FC<Props> = ({ people }) => {
+  const currentSlug = useMatch('/people/:slug')?.params.slug;
```

Comment:
You can get the slug with useParams hook.
`const {slug} = useParams()`

Reply:
What do I pass inside useParams?

Reply:
You don't pass anything.  It return an object that has your parameters in that page so lets say you have `/users/:userid/:todoid` when you destructer it you get `const = { userid, todoid } = useParams()`
But be aware it returns string so if it is an id don;t forget to convert it to number.